### ISSUE TITLE
rebase

### ISSUE THREAD

**Issue by k4lizen** [omentic/helix-ext#73]
more than 3 months has passed since the last release, plz rebase

**Comment by omentic** [comment]
i'll check it out post-midterms. there are some other patches i want to pull in too

**Comment by neel04** [comment]
@omentic `25.01` is out :) would appreciate a fresh update

**Comment by omentic** [comment]
rebased on `25.01.1`. sorry for the delay!

i removed the file explorer, as there is now a new file picker merged upstream. i'll probably not rebase it as i didn't use it much at all to begin with, but if someone really prefers it over the new file picker and rebases it i'll merge it (if someone does so though probably best to start from the `24.03` version here)

also if anyone has requests for patches to merge lmk

**Comment by k4lizen** [comment]
Yay thanks! 🥳 

> i removed the file explorer, as there is now a new file picker merged upstream.

aaaa that was the main reason I used this ;;
(and it's nice that it's on the AUR)

From release notes: 
> Simple pickers such as the file picker (<space>f) which only need one column are unchanged.

Do you use something else to emulate the file explorer UX (yazi, ranger, nnn)?

**Comment by omentic** [comment]
i found myself not using it at all. there are a lot of people doing different things in here though: https://github.com/helix-editor/helix/discussions/8314 (some of these approaches look quite good)

i have a shortcut for spawning new terminals in the current context with `mod+shift+enter` so that's what i usually did for opening new files and such. alongside using a graphical file manager for manipulating files

**Comment by k4lizen** [comment]
> i have a shortcut for spawning new terminals in the current context with mod+shift+enter so that's what i usually did for opening new files and such. 

yeah but that's not really viable for me when I have many buffers from the same project open, and opening then in different terminal windows brings up sync issues https://github.com/helix-editor/helix/issues/312

> there are a lot of people doing different things in here though: https://github.com/helix-editor/helix/discussions/8314 (some of these approaches look quite good)

I know I know, I set up the same thing for me with kitty + yazi (which is what I usually use for file managment), but it looks bad and is buggy https://github.com/helix-editor/helix/issues/11876 , the integrated file explorer has it's flaws too but at least it doesn't bug out on me

> i found myself not using it at all. 

I understand, I think I'll switch to upstream helix for a bit and try to get used to not having it, maybe I need to get used to using picker filters or something.

**Comment by justaskz** [comment]
There are so many terminal based editors but none is providing basic feature set found in app editors (sublime, vscode etc). That is file explorer, tabs, LSP support, plugins maybe. To make nvim work like basic sublime install even without plugins, you need to spend hours for researching and configuring stuff.

helix seemed as a promising project with many things coming out of the box. I hoped that missing things are in roadmap and will come in the near future. But it seems I am wrong :D

**Comment by k4lizen** [comment]
A file explorer has been merged to master! https://github.com/helix-editor/helix/pull/11285
